18 display rpaf code

kej-jay opened the pull request:
closes #18

## Changes
upon switching to the robot file tab in the web app there now is a code editor present which displays the code generated from the ssot in the database. 
Additionally there is a button to save changes to the cloud which can be used in the future to parse the code and upset it into the database. Currently the method for this already exists, but will only log the robotId and code to the console to display whoever is working on the #190 which variables can be used and where it should be implemented.

## Screenshots
<img width="1401" alt="image" src="https://user-images.githubusercontent.com/44369294/112945278-5ed0c800-9134-11eb-8fcf-113915adbef6.png">



## Checklist before merge

**Developer's responsibilities**
* [ ] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [ ] **Pull request build** has passed
* [ ] **tested locally** (in at least chrome & firefox if frontend)
* [ ] updated the **documentation**
* [ ] added **tests** where necessary

SanJSp commented on client/src/components/pages/RobotFile/RobotFile.jsx:
```diff
@@ -1,21 +1,79 @@
+   */
+  const onSaveToCloud = async () => {
+    const robotId = JSON.parse(sessionStorage.getItem('robotId'));
+    console.log(robotId);
+    console.log(code);
```
maybe to fix the linter issues:
```suggestion
    console.warn(robotId,code);
```

kej-jay replied:
But it is not really a warning, so I wouldn't want to use that.

SanJSp reviewed (changes requested):
Code wise it looks fine. But in the issue it is stated, that the code should not be editable. Are you aware of that?

kej-jay commented:
> Code wise it looks fine. But in the issue it is stated, that the code should not be editable. Are you aware of that?

Because of the Parser connection issue #1990 I built it so that it could be edited and also added the button already those were not required, but knowing that it will be used in a matter of days or even only hours I added them.
